api/wallet/spec/paths/withdrawals.yaml

```
@@ -44,75 +45,74 @@ get:
       required: false
     - name: createdAtFrom
       in: query
-      description: Дата создания с
+      description: Creation date from
       type: string
       format: date-time
       required: false
     - name: createdAtTo
       in: query
-      description: Дата создания до
+      description: Creation date to
       type: string
       format: date-time
       required: false
-    - $ref: '#/parameters/amountFrom'
-    - $ref: '#/parameters/amountTo'
-    - $ref: '#/parameters/currencyID'
-    - $ref: '#/parameters/limit'
+    - $ref: "#/parameters/amountFrom"
+    - $ref: "#/parameters/amountTo"
+    - $ref: "#/parameters/currencyID"
+    - $ref: "#/parameters/limit"
     - x-rebillyMerge:
-      - name: continuationToken
-        in: query
-        required: false
-      - $ref: '#/definitions/ContinuationToken'
+        - name: continuationToken
+          in: query
+          required: false
+        - $ref: "#/definitions/ContinuationToken"
   responses:
-    '200':
-      description: Результат поиска
+    "200":
+      description: Search result
       schema:
         type: object
         properties:
           continuationToken:
             x-rebillyMerge:
-              - $ref: '#/definitions/ContinuationToken'
+              - $ref: "#/definitions/ContinuationToken"
           result:
-            description: Найденные выводы
+            description: Withdrawals found
             type: array
             items:
-              $ref: '#/definitions/Withdrawal'
-    '400':
-      $ref: '#/responses/BadRequest'
-    '401':
-      $ref: '#/responses/Unauthorized'
-
+              $ref: "#/definitions/Withdrawal"
+    "400":
+      $ref: "#/responses/BadRequest"
+    "401":
+      $ref: "#/responses/Unauthorized"
 post:
   operationId: createWithdrawal
-  summary: Запустить вывод средств
+  summary: Start funds withdrawal
   tags:
     - Withdrawals
   parameters:
-    - $ref: '#/parameters/requestID'
-    - $ref: '#/parameters/deadline'
+    - $ref: "#/parameters/requestID"
+    - $ref: "#/parameters/deadline"
     - name: withdrawal
-      description: Данные вывода
+      description: Withdrawal data
       in: body
       required: true
       schema:
-        $ref: '#/definitions/WithdrawalParameters'
+        $ref: "#/definitions/WithdrawalParameters"
   responses:
-    '202':
-      description: Вывод запущен
+    "202":
+      description: Withdrawal started
       headers:
         Location:
-          description: URI запущенного вывода
+          description: URI of withdrawal started
```

Будто бы здесь подменяется объект
```suggestion
          description: URI of started withdrawal
```